src/frontends/pytorch/src/op/sum.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_sum(NodeContext& context) {
+    bool keep_dims = false;
+    ov::Output<ov::Node> axes;
+    Output<Node> cast;
+    FRONT_END_OP_CONVERSION_CHECK(context.get_input_size() >= 1, "Operation has no inputs.");
+    FRONT_END_OP_CONVERSION_CHECK(!context.input_is_none(0), "Input should not be None.");
+    auto data = context.get_input(0);
+    if (context.get_input_size() == 1 || context.input_is_none(1)) {
```

Comment:
No need to verify input_size and is_none. `input_is_none` does this check internally.

Reply:
removed